Formula/wiredtiger.rb

```diff
@@ -21,9 +22,12 @@ class Wiredtiger < Formula
 
   uses_from_macos "zlib"
 
+  # Workaround to build on ARM with system type 'arm-apple-darwin*'
+  # Remove in next release as build system is changing to CMake
+  patch :DATA
```

Comment:
Although Apple ARM doesn't seem to be supported, compile is successful using ARM code for Linux.